src/sentry/features/base.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import absolute_import
+
+__all__ = ['Feature', 'OrganizationFeature']
+
+
+class Feature(object):
+    def __init__(self, name):
+        self.name = name
+
+    def has(self, actor):
+        """
+        A feature may return one of three values:
+
+        - True: the feature is enabled for actor
+        - False: the feature is not enabled for actor
+        - None: defer
+        """
+        return None
+
+
+class OrganizationFeature(object):
```

Comment:
The thinking behind this is that it keeps the signature of has() the same everywhere (vs an arbitrary number of arguments), and we can just implement attributes for any objects we may want to pass.